codersdk/oauth2.go

```diff
@@ -21,10 +21,10 @@ const (
 )
 
 type OAuth2ProviderApp struct {
-	ID          uuid.UUID `json:"id" format:"uuid"`
-	Name        string    `json:"name"`
-	CallbackURL string    `json:"callback_url"`
-	Icon        string    `json:"icon"`
+	ID           uuid.UUID `json:"id" format:"uuid"`
+	Name         string    `json:"name"`
+	RedirectURIs []string  `json:"redirect_uris"`
```

Comment:
As I understand, you could only get this response if the OAuth2 provider was enabled, which was only available on development builds and not release builds. So not a breaking change, correct?

Reply:
Yes, either a dev build or if one actively enables the oauth2 experiment